pkg/scraper/jobcenter_scraper.go

```diff
@@ -1,70 +1,99 @@
 package scraper
 
 import (
-	"errors"
 	"fmt"
 	"regexp"
+	"strconv"
+	"strings"
+	"sync"
 
 	"github.com/PuerkitoBio/goquery"
 	"github.com/b-open/jobbuzz/pkg/model"
+	"github.com/pkg/errors"
 )
 
 const (
-	pageSize = 30
+	pageSize = 200
 
 	jobcenterUrl = "https://www.jobcentrebrunei.gov.bn"
 )
 
-func ScrapeJobcenter() ([]*model.Job, error) {
-
-	jobs := []*model.Job{}
+var jobcenterJobs = []*model.Job{}
```

Comment:
Does it not work if you declare this within the scope of the scrape function and pass it as a pointer to the go routine function?

Reply:
I changed my approach, I'm using a scraper struct to hold the wg and slice